config/feed/README.md

```diff
@@ -0,0 +1,128 @@
+# Feed Configuration
+
+Welcome to the technical documentation of the Feed Configuration. In this
+document we'll introduce the historical context, provide a glossary of terms,
+delve into the high-level goals of the configurations, and describe any present
+constraints.
+
+## Historical Documents
+
+- The
+  [Articles::Feeds::WeightedQueryStrategy](https://github.com/forem/forem/blob/de2edee07d824a34e5c5445d455b1f8086bd127d/app/services/articles/feeds/weighted_query_strategy.rb)
+  is the precursor to the more robust feed configuration. It provides hints as
+  to how we'll structure the feed configuration.
+- [These Are the [Feed] Levers I Know I Know - Forem Team 🌱](https://forem.team/jeremy/these-are-the-feed-levers-i-know-i-know-293m)
+- [Creating a Catalog of Feed Levers and Configurations - Forem Team 🌱](https://forem.team/jeremy/creating-a-catalog-of-feed-levers-and-configurations-nnk)
+- On <2022-04-11 Mon>, Jeremy, Jennie, Ella, and Michael T met over Zoom to talk
+  through the current feed.
+
+## Glossary
+
+- **_Feed Experiment_:** A _feed experiment_ is the current _feed strategies_
+  current configurations we're using to generate feeds
```

Comment:
```suggestion
- **_Feed Experiment_:** A _feed experiment_ is the currently implemented set of _feed strategies_ -
  current configurations we're using to generate feeds
```

When I first read this and the next bullet point I wasn't too clear on the difference between _feed_experiment_ vs _feed_strategies_. I'm not sure if my suggestion captures the distinction we're trying to make here properly, so feel free to ignore if it doesn't seem right 😄